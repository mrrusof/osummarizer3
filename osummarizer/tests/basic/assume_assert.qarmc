'ctx_f1_int->unit'(A_F1_V) :- A_F1_V=_.
'f1_int->unit'(X2) :- X2>0, X2>1, 'ctx_f1_int->unit'(X2).
X2>0 :- X2>1, 'ctx_f1_int->unit'(X2).
