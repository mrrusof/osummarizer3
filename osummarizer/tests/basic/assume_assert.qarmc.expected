'ctx_f1_int->unit'(A_F1_V) :- A_F1_V=_.
X2>0 :- X2>1, 'ctx_f1_int->unit'(X2).
'f1_int->unit'(X2, RET_F1) :- X2>0, X2>1, 'ctx_f1_int->unit'(X2).
